docs/core-plugins/eslint.md

```diff
@@ -17,7 +17,7 @@
     --max-warnings       specify number of warnings to make build failed (default: Infinity)
   ```
 
-  Lints and fixes files. If no specific files are given, it lints all files in `src` and `tests`.
+  Lints and fixes files. If no specific files are given, it lints all files in `src` and `tests`, as well as all Javascript files in the root directory (these are most often config files such as `babel.config.js` or `.eslintrc.js`).
```

Comment:
Javascript => JavaScript

Reply:
The sentence you added is good, why did you delete it? 🤣  @Robin-Hoodie

Reply:
My bad, I misinterpreted https://github.com/vuejs/vue-cli/pull/6199#pullrequestreview-566003817 to mean that my original sentence should also be removed